cinema/management/commands/wait_for_db.py

```
@@ -0,0 +1,31 @@
+import os
+import time
+
+import psycopg2
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        for i in range(1, 21):
+            self.stdout.write(f"Connecting to DB, try #{i} out of 20:")
+            try:
+                conn = psycopg2.connect(
+                    f"dbname={os.environ.get('POSTGRES_DB')} "
+                    f"user={os.environ.get('POSTGRES_USER')} "
+                    f"host={os.environ.get('POSTGRES_HOST')} "
+                    f"password={os.environ.get('POSTGRES_PASSWORD')} "
+                    f"port={os.environ.get('POSTGRES_PORT')}"
```

Using `os.environ.get()` can hide configuration issues. If any of these environment variables are not set, `.get()` will return `None`, leading to a connection string with values like `'dbname=None'`. This will cause the connection to fail without a clear error message about the missing variable. It's better to use `os.environ[...]` like in `settings.py` to ensure the application fails fast with a clear `KeyError` if a required environment variable is missing.